buildenv

```diff
@@ -0,0 +1,184 @@
+# Update bump details accordingly. Use bump check to confirm.
+# bump: postgres-version /POSTGRES_VERSION="(.*)"/ https://github.com/postgres/postgres.git|semver:*
+# POSTGRES_VERSION="V.R.M" # Specify a stable release
+# export ZOPEN_STABLE_TAG="v${POSTGRES_VERSION}"
+export ZOPEN_DEV_TAG="REL_18_BETA1"
+
+###
+### Build control file for the postgres port
+###
+
+###
+### Required Environment Variables
+###
+
+## Specify the default build line, either 'DEV' or 'STABLE' (default: STABLE)
+export ZOPEN_BUILD_LINE="DEV"
+
+## Required IF ZOPEN_BUILD_LINE='STABLE'
+export ZOPEN_STABLE_URL=""  # Specify the stable build URL (either git or tarball)
+export ZOPEN_STABLE_DEPS="curl git make m4 perl autoconf automake help2man texinfo xz zlib openssl expat gettext gzip tar coreutils zoslib diffutils ncurses bash sed libpcre2 tar check_python gawk" # Specify the stable build dependencies.
+
+## Required IF ZOPEN_BUILD_LINE='DEV'
+export ZOPEN_DEV_URL="https://github.com/postgres/postgres.git"   # Specify the dev build URL
+export ZOPEN_DEV_DEPS="curl git make m4 perl autoconf automake help2man texinfo xz zlib openssl expat gettext gzip tar coreutils zoslib diffutils ncurses bash sed libpcre2 tar check_python gawk pkgconfig bison flex"  # Specify the dev build dependencies
+
+## Specify a space-delimited list of applicable categories.
+## See tool_categories.txt for valid options
+export ZOPEN_CATEGORIES="database"
+
+## Runtime zopen tool dependencies to be installed alongside the tool.
+export ZOPEN_RUNTIME_DEPS="zlib libxml2 libxslt lz4 systemd tcl python perl"
```

Comment:
If they're statically linked, then we don't need to add the libraries are runtime dependencies